template/daemonset_openshift.yaml

```diff
@@ -172,3 +172,23 @@ spec:
       - name: cnibin
         hostPath:
           path: /opt/cni/bin
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: bridge-filtering
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+readOnlyRootFilesystem: false
+allowHostIPC: false
+allowHostNetwork: true
```

Comment:
I don't think this is needed - I don't even think the daemonset spec is setting it.